apps/dashboard/services/graphql/queries/get-transactions.ts

```diff
@@ -0,0 +1,202 @@
+import { gql } from "@apollo/client";
+
+import { apollo } from "..";
+import {
+  GetFirstTransactionsDocument,
+  GetFirstTransactionsQuery,
+  GetPaginatedTransactionsDocument,
+  GetPaginatedTransactionsQuery,
+} from "../generated";
+
+// TODO remove not used fields
+gql`
+  query GetPaginatedTransactions($first: Int, $after: String, $before: String) {
```

Comment:
why are 2 queries needed for transactions? it has not be needed to declare both in other repo afaik.